app/views/shared/tas/_warren.html.slim

```diff
@@ -0,0 +1 @@
+| Computer Science student here at Cal.
```

Comment:
Weak.

In all seriousness though I think having a longer bio will at least make you slightly more approachable for OH.